src/pages/Introduction/Introduction.scss

```diff
@@ -1,5 +1,6 @@
 .introduction {
     height: 100vh;
```

Comment:
Adjust the background color to maintain consistency with the updated color scheme. Ensure the color choice complements the design theme. [View Reference](https://github.com/raiaqt)